src/ekat/logging/ekat_log_mpi.hpp

```diff
@@ -15,35 +15,20 @@
 // methods.
 namespace ekat {
 
-/* Console log ignores MPI Rank; all ranks will write to the console without rank ids.
-  File logs (if used) will still have their rank id in their filenames.
-*/
-struct LogIgnoreRank {
-  // leave sinks alone
-  static void update_sinks(std::vector<spdlog::sink_ptr>& sinks) {}
-
-  // leave name alone
-  static std::string name_with_rank(const std::string& name) {
-    return name;
-  }
-};
-
-
 /* Suppress console output from ranks > 0.  All ranks still write to files, if
   file logging is enabled.
 */
 struct LogOnlyRank0 {
-  // erase the console sink for all ranks != 0
-  static void update_sinks(std::vector<spdlog::sink_ptr>& sinks) {
-    const Comm mpicomm(MPI_COMM_WORLD);
+  // nullify the console sink for all ranks != 0
```

Comment:
Question: should we nullify also the file sink? If the MPI policy is different for C and F sinks, then we should have a policy for each.

Reply:
I removed the MPI Policy template parameter, and replaced that functionality with two class methods, `console_output_rank0_only` and `file_output_rank0_only` that can be called independently of each other.